fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResource.java

```diff
@@ -498,12 +499,14 @@ public String retrieveAll(@Context final UriInfo uriInfo,
             loanBasicDetails.getPageItems().forEach(i -> {
                 if (i.getSummary() != null) {
                     Collection<DisbursementData> disbursementData = this.loanReadPlatformService.retrieveLoanDisbursementDetails(i.getId());
+                    List<LoanTransactionData> capitalizedIncomeTransactions = this.loanReadPlatformService
```

Comment:
We dont need the transactions. all the balance can be found in `m_loan_capitalized_income_balance`